### PR TITLE
profile

### PR DESCRIPTION
**Pull request by CarlosFarrica**
change the word porfile into profile

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The following table provides a helpful quick reference to feature availability b
 
 <sup>6</sup> Only contacts in default contacts folder. Access to other contacts folders or sub-folders is not supported.
 
-<sup>7</sup> Teams honors the [Outlook on the web mailbox policy](/powershell/module/exchange/client-access/set-owamailboxpolicy) setting that's configured by tenant admins to control whether users can change their profile picture. If the **-SetPhotoEnabled** setting is turned off in the policy, users cannot add, change, or remove their profile picture, so the porfile picture wont be synced to teams if the admin changes the photo.
+<sup>7</sup> Teams honors the [Outlook on the web mailbox policy](/powershell/module/exchange/client-access/set-owamailboxpolicy) setting that's configured by tenant admins to control whether users can change their profile picture. If the **-SetPhotoEnabled** setting is turned off in the policy, users cannot add, change, or remove their profile picture, so the profile picture wont be synced to teams if the admin changes the photo.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:62]:
```suggestion
<sup>7</sup> Teams honors the [Outlook on the web mailbox policy](/powershell/module/exchange/client-access/set-owamailboxpolicy) setting that's configured by tenant admins to control whether users can change their profile picture. If the **-SetPhotoEnabled** setting is turned off in the policy, users cannot add, change, or remove their profile picture, so the profile picture won't be synced to teams if the admin changes the photo.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @CarlosFarrica, kindly commit the suggested change to correct the additional minor typo. Many thanks.

File: Teams/Exchange-Teams-interact.md (hunk 1 above)
Comment (CarlosFarrica) [reply Teams/Exchange-Teams-interact.md:62]: Agree

**Comment by CarlosFarrica** [comment]
Please proceed with the commit.

**Comment by JohanFreelancer9** [comment]
@scanum - These typos have been fixed in PR #7634 - could you please close this PR? Thanks.

**Pull request closed without merging**